static/svg/luva-inkscape.svg

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
I just liked the choice of editor. As I'd said above, I want to generate these by hand to allow that fine control.